lib/main_common.pm

```diff
@@ -1539,7 +1543,7 @@ sub load_extra_tests_docker {
     return unless is_sle('12-SP3+') || !is_sle;
     loadtest "console/docker";
     loadtest "console/docker_runc";
-    if (is_sle('12-SP3+')) {
+    if (is_sle('12-SP3+') && is_sle('<15')) {
```

Comment:
why you want to block docker_image test in SLE 15 ? @SergioAtSUSE  , @drpaneas might be interesting for you

Reply:
actually it's not my change, I will do rebase later